fastlane.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'terminal-notifier', '>= 1.6.2', '< 2.0.0' # macOS notifications
   spec.add_dependency 'terminal-table', '>= 1.4.5', '< 2.0.0' # Actions documentation
   spec.add_dependency 'plist', '>= 3.1.0', '< 4.0.0' # Needed for set_build_number_repository and get_info_plist_value actions
+  spec.add_dependency 'CFPropertyList', '~> 2.3.3' # Needed to be able to read binary plist format
```

Comment:
I wouldn't be too strict about this, maybe have `'>= 2.3', '< 3.0.0'` instead. If `xcodeproj` bumps their dependency, it would result in unresolvable dependencies otherwise